include/Customer.h

```diff
@@ -1,19 +1,41 @@
 #pragma once
 
+//** KOOS - I would probably encourage the use of library instead of source
```

Comment:
Also fair; this computer is a bit old and I didn't want to set up a whole project. It's small enough to include as a single file, so it didn't seem like the worst for simplicity.